packages/test-utils/src/reporter.ts

```diff
@@ -264,7 +262,7 @@ class ReplayReporter<TRecordingMetadata extends UnstructuredMetadata = Unstructu
     schemaVersion: string,
     config?: ReplayReporterConfig<TRecordingMetadata>
   ) {
-    setUserAgent(`${pkgJson.name}/${pkgJson.version}`);
+    // setUserAgent(`${pkgJson.name}/${pkgJson.version}`);
```

Comment:
@bvaughn https://github.com/replayio/replay-cli/pull/575 moved `test-utils` to a dev dependencies of other packages - so it's now a bundled dependency of them. We should add `setUserAgent` to all of those `test-utils` dependents, right?

Reply:
Yeah that would make more sense now that test-utils is inlined

Reply:
IMO we should just delete this line though rather than commenting it out